lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -54,46 +54,35 @@ var DataObservatoryMultipleMeasures = Node.create(TYPE, PARAMS, { cache: true, l
 
 module.exports = DataObservatoryMultipleMeasures;
 
+DataObservatoryMultipleMeasures.prototype._composeObsMetaParams = function() {
+    var params = this.numerators.map(function (numerator, index) {
+        return {
+            numer_id: numerator,
+            denom_id: (this.denominators && this.denominators[index]) ?
+                this.denominators[index] : null,
+            normalization: this.normalizations[index],
+            geom_id: (this.geom_ids && this.geom_ids[index]) ?
+                this.geom_ids[index] : null,
+            numer_timespan: (this.numerator_timespans && this.numerator_timespans[index]) ?
+                this.numerator_timespans[index] : null
+        };
+    }.bind(this));
+
+    return JSON.stringify(params);
```

Comment:
This will generate a string version of an array that in combination with the template `(\'[{{=it.obsMetaParams}}]\')::JSON` will end having an array inside an array.

```
('[[{"numer_id":"es.ine.t1_1","denom_id":null,"normalization":"denominated","geom_id":"es.ine.the_geom","numer_timespan":"2011"}]]')::JSON
```

vs

```
('[{"numer_id":"es.ine.t1_1","denom_id":null,"normalization":"denominated","geom_id":"es.ine.the_geom","numer_timespan":"2011"}]')::JSON
```

That subtle difference will make the query to fail to find any metadata.